doc/xeplist.xml

```diff
@@ -0,0 +1,99 @@
+<xep-infos><xep accepted="true"><number>130</number><title>Waiting Lists</title><abstract>This document defines an XMPP protocol extension that enables a user to add a non-IM user to a waiting list and be informed when the contact creates an IM account.</abstract><type>Historical</type><status>Deprecated</status><approver>Council</approver><shortname>waitinglist</shortname><last-revision><date>2021-03-04</date><version>1.4.1</version><initials>mw</initials><remark>Cross-document editorial adjustments for inclusive language.</remark></last-revision><sig>Standards</sig></xep><xep accepted="true"><number>223</number><title>Persistent Storage of Private Data via PubSub</title><abstract>This specification defines best practices for using the XMPP publish-subscribe extension to persistently store private information such as bookmarks and client configuration options.</abstract><type>Informational</type><status>Active</status><approver>Council</approver><last-revision><date>2018-03-28</date><version>1.1</version><initials>jwi</initials><remark>Make discovery of support mandatory, add security considerations (in reaction to CVE-2018-6591).</remark></last-revision><sig>Standards JIG</sig></xep><xep accepted="true"><number>233</number><title>XMPP Server Registration for use with Kerberos V5</title><abstract>This specification defines the Kerberos principal name of an XMPP server. It also details a method by which a connecting client can determine this Kerberos principal name when authenticating using the "GSSAPI" SASL mechanism.</abstract><type>Standards Track</type><status>Draft</status><approver>Council</approver><shortname>kerberos5</shortname><last-revision><date>2017-03-16</date><version>1.0.0</version><initials>XEP Editor (ssw)</initials><remark>Move to draft by order of the XMPP Council.</remark></last-revision><sig>Standards</sig><lastcall>2017-03-01</lastcall></xep><xep accepted="true"><number>257</number><title>Client Certificate Management for SASL EXTERNAL</title><abstract>This specification defines a method to manage client certificates that can be used with SASL External to allow clients to log in without a password.</abstract><type>Standards Track</type><status>Deferred</status><approver>Council</approver><last-revision><date>2012-07-18</date><version>0.3</version><initials>tpa</initials><remark>* Removed the dependency on .
```

Comment:
how does updating of this file work?

Reply:
The latest version is fetched now before generating the documentation.